doc/modules/model_evaluation.rst

```diff
@@ -442,6 +449,45 @@ In the multilabel case with binary label indicators: ::
     for an example of accuracy score usage using permutations of
     the dataset.
 
+.. _top_k_accuracy_score:
+
+Top-k Accuracy score
+--------------------
+
+The :func:`top_k_accuracy_score` function is a generalization of
+:func:`accuracy_score`. The difference is that a prediction is considered
+correct as long as the true label is associated with one of the ``k`` highest
+predicted scores. :func:`accuracy_score` is the special case of `k = 1`.
+
+The function covers the binary and multiclass classification cases but not the
+multilabel case.
+
+If :math:`\hat{y}_{i,j}` is the predicted class corresponding to :math:`j`-th
```

Comment:
We should rather adopt the notation used below for multilabel ranking metrics, I think. It's very confusing to have `\hat{y}` here indicate a matrix.